looker_blueprints/download_dashboard.py

```diff
@@ -0,0 +1,113 @@
+import argparse
+import sys
+import time
+import os
+import shipyard_utils as shipyard
+from looker_sdk import models
+from looker_sdk.error import SDKError
+try:
+    import helpers
+except BaseException:
+    from . import helpers
+try:
+    import exit_codes as ec
+except BaseException:
+    from . import exit_codes as ec
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--base-url', dest='base_url', required=True)
+    parser.add_argument('--client-id', dest='client_id', required=True)
+    parser.add_argument('--client-secret', dest='client_secret', required=True)
+    parser.add_argument('--dashboard-id', dest='dashboard_id', required=True)
+    parser.add_argument('--output-width', dest='output_width', required=True)
+    parser.add_argument('--output-height', dest='output_height', required=True)
+    parser.add_argument('--destination-file-name', dest='dest_file_name', required=True)
+    parser.add_argument('--destination-folder-name', dest='dest_folder_name', required=False)
+    parser.add_argument('--file-type', dest='file_type',
+                        choices=['pdf', 'png', 'jpg'],
+                        type=str.lower,
+                        required=True)
+    args = parser.parse_args()
+    return args
+
+
+def download_dashboard(sdk, dashboard_id, width, height, file_format):
+    """Download specified dashboard using ID
+    
+    Returns:
+        result: raw binary data of the dashboard
+    """
+    dash_id = dashboard_id
```

Comment:
Seems redundant? Just use dashboard_id in the following function?

Reply:
This is done